src/test/java/picard/analysis/TheoreticalSensitivityTest.java

```diff
@@ -216,13 +216,117 @@ public void testHetSensTargeted(final double expected, final File metricsFile) t
         final Histogram depthHistogram = histograms.get(0);
         final Histogram qualityHistogram = histograms.get(1);
 
-        final double [] depthDistribution = TheoreticalSensitivity.normalizeHistogram(depthHistogram);
-        final double [] qualityDistribution = TheoreticalSensitivity.normalizeHistogram(qualityHistogram);
+        final double[] depthDistribution = TheoreticalSensitivity.normalizeHistogram(depthHistogram);
+        final double[] qualityDistribution = TheoreticalSensitivity.normalizeHistogram(qualityHistogram);
 
         final int sampleSize = 1_000;
         final double logOddsThreshold = 3.0;
 
         final double result = TheoreticalSensitivity.hetSNPSensitivity(depthDistribution, qualityDistribution, sampleSize, logOddsThreshold);
         Assert.assertEquals(result, expected, tolerance);
     }
+
+    @DataProvider(name = "TheoreticalSensitivityConstantDepthDataProvider")
+    public Object[][] fractionalAlleleSensDataProvider() {
+        final File wgsMetricsFile = new File(TEST_DIR, "test_Solexa-332667.wgs_metrics");
+        final File targetedMetricsFile = new File(TEST_DIR, "test_25103070136.targeted_pcr_metrics");
+
+        //These magic numbers come from a separate implementation of the code in R.
+        return new Object[][]{
+                // expected sensitivity, metrics file, allele fraction, constant depth, sample size.
+                {1.00, wgsMetricsFile, .5, 30, 10000},
+                {1.00, wgsMetricsFile, .5, 30, 10000},
+                {0.78, targetedMetricsFile, .1, 30, 10000},
+                {0.26, targetedMetricsFile, 0.1, 10, 10000}
```

Comment:
I need to add a test for high depth and low allele fraction.  That was actually the original purpose of this test...

Also, I need to make sure my claim about the magic numbers coming from R is true.